modules/olm-updating-sqlite-catalog-to-a-new-opm-version.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-managing-custom-catalogs.adoc
+
+:_content-type: PROCEDURE
+[id="olm-updating-sqlite-catalog-to-a-new-opm-version_{context}"]
+= Rebuilding SQLite database catalogs with a more recent version of the opm CLI tool
+
+You can rebuild your SQLite database catalog with a more recent version of the `opm` CLI tool that is released with your version of {product-title}.
+
+.Prerequisites
+
+* An SQLite database catalog source
+* Cluster administrator permissions
+
+.Procedure
+
+* Run the following command to rebuild your catalog with a more recent version of the `opm` CLI tool:
++
+[source,terminal]
+----
+$ opm index add --binary-image \
+  registry.redhat.io/openshift4/ose-operator-registry:<ocp_version_tag> \ <1>
+  --from-index <your_registry_image> \
+  --bundles "" -t \<your_registry_image>
+----
+<1> Specify the version tag of your OpenShift Container Platform version, such as `v4.12`. You must use `v4.11` or later.
```

Comment:
@joelanford Does your comment about the version of `opm` on the workstation matching the OCP version tag also apply here?

Reply:
IIB uses an upstream version (generally pretty recent) for the `opm index add` command even if the binary image is older. I'm honestly not sure what the best guidance is here. It seems like aligning with the OCP version tag should be a safe bet. Maybe @awgreene could provide some input here as well.

Reply:
I agree that aligning with OCP version tag is probably the best path forward given that IIB may swap opm versions at their discression.

If we had a time machine, I think we should have created a way to identify which version of OPM (git commit or tag) was used to build the catalogSource. This may be less of a problem with the recent move to FBC catalogs, but I imagine that tracking this information somewhere may pay dividends going forward if we ever make a change to the format that isn't easily inferred from looking at the structure of the FBC.